### PR TITLE
Minor fix for calibrate tool

### PR DESCRIPTION
**Pull request by andrey-churkin**
### Changes

The initialization of the model evaluator was fixed.

### Reason for changes

Some models failed.

### Related tickets

N/A

### Tests

N/A

### CODE REVIEW

**Comment by andrey-churkin** [comment]
Build #9019

**Comment by andrey-churkin** [comment]
run openvino pre-commit tests

**Comment by andrey-churkin** [comment]
run pytorch pre-commit tests

File: tests/openvino/tools/calibrate.py [hunk 1]
```diff
@@ -389,7 +389,7 @@ def quantize_model_with_accuracy_control(xml_path: str,
                                          quantization_parameters):
     ov_model = ov.Core().read_model(xml_path, bin_path)
     model_evaluator = create_model_evaluator(accuracy_checcker_config)
-    model_evaluator.load_network([{'model': ov_model}])
```
Comment (alexsu52) [inline tests/openvino/tools/calibrate.py:392]: Why did this cause the model to fail?

**Review by alexsu52 — APPROVED** [review]

**Pull request merged**